lib/onelogin/ruby-saml/utils.rb

```diff
@@ -32,7 +32,7 @@ class Utils
           (\d+)W                    # 8: Weeks
         )
       $)x.freeze
-      UUID_PREFIX = '_'
+      UUID_PREFIX = +'_'
```

Comment:
Constants should remain frozen. Instead, call `UUID_PREFIX.dup` where it is used, if necessaty.

Reply:
It's used with `.replace` which won't work. Reassigning the constant is "bad".
I'm replacing it with a variable, but should we keep it for compatibility, or remove it and bump a major version?